packages/datetime/src/datePickerCaption.tsx

```diff
@@ -36,17 +36,17 @@ export class DatePickerCaption extends React.PureComponent<IDatePickerCaptionPro
     private handleYearSelectChange = this.dateChangeHandler((d, year) => d.setFullYear(year), this.props.onYearChange);
 
     public render() {
-        const { date, locale, localeUtils, minDate, maxDate } = this.props;
+        const { date, locale, localeUtils, minDate, maxDate, months } = this.props;
         const minYear = minDate.getFullYear();
         const maxYear = maxDate.getFullYear();
         const displayMonth = date.getMonth();
         const displayYear = date.getFullYear();
 
         // build the list of available months, limiting based on minDate and maxDate as necessary
-        const months = localeUtils.getMonths(locale);
+        const monthsLabels = months ? months : localeUtils.getMonths(locale);
```

Comment:
this can actually be done nicely as a deconstructor default, and you'll avoid the variable rename:

```js
const { ..., months = localeUtils.getMonths(locale) } = this.props;
```

Reply:
💯